services/pkgreview/package-rating.service.js

```diff
@@ -74,26 +73,22 @@ module.exports = class PkgreviewRating extends BaseJsonService {
       schema,
       url: `https://pkgreview.dev/api/v1/${pkgManager}/${pkgSlug}`,
       errorMessages: {
-        400: 'package not found',
+        400: 'bad request',
```

Comment:
What circumstances (namely, what user-provided params to Shields.io) would result in Shields receiving a 400 response?

Reply:
For unsupported `pkgManager`, the API responds with `400 Bad Request`.

For eg. `https://pkgreview.dev/api/v1/lol/react`, i.e. `/lol/react`

Reply:
The `Bad Request` from `pkgreview.dev API` is now handled with your pattern syntax: `/:pkgMananger(npm)`... So, yes, it is no more needed.

Removed the `400: 'bad request'` error. :)

Reply:
As you add support for new package managers to the service, drop us a note (or even better a PR 😄) so that we can update the route accordingly to accept those additional package managers